treeherder/services/elasticsearch/connection.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+from elasticsearch import Elasticsearch
+
+url = settings.ELASTICSEARCH_URL
+if url:
+    es_conn = Elasticsearch(settings.ELASTICSEARCH_URL)
```

Comment:
`s/settings.ELASTICSEARCH_URL/url/` (sorry for not spotting before)

Or possibly one step further: `es_conn = Elasticsearch(url) if url else None`
(up to you; guess with Lazy might change soon anyway)